outport/process/transactionsfee/transactionsFeeProcessor_test.go

```diff
@@ -470,3 +471,51 @@ func TestPutFeeAndGasUsedScrWithRefundNotForInitialSender(t *testing.T) {
 	require.Equal(t, big.NewInt(0), scr.GetFeeInfo().GetFee())
 	require.Equal(t, uint64(0), scr.GetFeeInfo().GetGasUsed())
 }
+
+func TestPutFeeAndGasUsedScrWithRefund(t *testing.T) {
+	t.Parallel()
+
+	txHash := []byte("e3cdb8b4936fdbee2d3b1244b4c49959df5f90ada683d650019d244e5a64afaf")
+	scrWithRefund := []byte("scrWithRefund")
+
+	initialTx := &outportcore.TxInfo{Transaction: &transaction.Transaction{
+		Nonce:    1004,
+		GasLimit: 60_000_000,
+		GasPrice: 1000000000,
+		SndAddr:  []byte("erd1s8jr8e8hsvv7c9ehmshcjlpzf9ua5l50qeswa8feshrp6xlz9c7quacmtx"),
```

Comment:
Do we have to use real addresses here?
I think we had a discussion before about non using real address, or limiting the usage to as low as possible

Reply:
as long as they are brand-new addresses, I think it's ok. We are a public blockchain, anyway :)